pitmutationmate/src/main/kotlin/com/amos/pitmutationmate/pitmutationmate/activity/LaunchActivity.kt

```diff
@@ -18,6 +23,24 @@ class LaunchActivity : ProjectActivity {
             val buildFiles = BuildSystemUtils.getProjectBuildFiles(project)
             thisLogger().debug("Detected buildFiles to analyze with PluginCheckerService on project open: $buildFiles")
             ApplicationManager.getApplication().invokeLater { pluginChecker.checkPlugins(buildFiles) }
+            // Open test results on loading project if reports are present
+            ApplicationManager.getApplication().invokeLater {
+                val coverageReport = MutationResultService(project).updateLastMutationResult()?.coverageReports?.first()
+                val mutationReport = MutationResultService(project).updateLastMutationResult()?.mutationResults?.first()
```

Comment:
This is not how a service is supposed to be used:
```
val resultService = file.project.service<MutationResultService>()
```

The way you use it will result in multiple Services being created when we always only want one